mitiq/shadows/post_processing.py

```diff
@@ -0,0 +1,141 @@
+from typing import Tuple, List, Union, Any
+
+import numpy as np
+
+PAULI_MAP = {"X": 0, "Y": 1, "Z": 2}
+
+
+def snapshot_state(
+    b_list: List[Union[int, float]], u_list: List[str]
+) -> np.ndarray[Any, Any]:
+    """
+    Implement a single snapshot state reconstruction,
+
+    Args:
+        b_list (array): The list of classical outcomes for the snapshot.
+        u_list (array): Indices for the applied Pauli measurement.
```

Comment:
Two things:

1. No need to add the type in the argument description. In fact it looks like it's losing the type-hint information when the docs are building the [API-doc](https://mitiq--1908.org.readthedocs.build/en/1908/apidoc.html#mitiq.shadows.post_processing.snapshot_state).
2. The description for `u_list` seems wrong. These aren't indices, but rather which Pauli measurement was performed.